### PR TITLE
feat(app): Add new font and clean code

### PR DESCRIPTION
**Pull request by ernum**

### CODE REVIEW

**Comment by Hymalide** [comment]
All the filters of the CP chart don't appear completely on my screen (Circle Size category is cut in the middle of the icon for the Google Search Volume)

**Comment by ernum** [comment]
> All the filters of the CP chart don't appear completely on my screen (Circle Size category is cut in the middle of the icon for the Google Search Volume)

Hmm, can you send a screenshot?

File: src/components/CirclePacking.js [hunk 1]
```diff
@@ -347,7 +347,7 @@ export default function CirclePacking(props) {
                 d3.select('#outerSvg')
                     .append('text').text(sortName)
                     .style("fill", font_color)
-                    .style("font", "10px montserrat")
+                    .style("font", "10px NeueHaasDisplay")
```
[the remark this answers is not in the document]
Comment (mansnym) [reply src/components/CirclePacking.js]: Maybe even use old font at least fur sort and size button texts... it jus seems very squashed and it probably wont align well with the buttons as they are right now if we just increase the font size. An option would be to increase the button sizes. Legend font definitely should be bigger tho I think

[the remark this answers is not in the document]
Comment (ernum) [reply src/components/CirclePacking.js]: I've updated the spacing between letters. How does it look now?

[the remark this answers is not in the document]
Comment (mansnym) [reply src/components/CirclePacking.js]: asdf

In general I think everything looks great now apart from the spacing in exercise name in detail list (see image)

[the remark this answers is not in the document]
Comment (ernum) [reply src/components/CirclePacking.js]: I've looked into and and don't think it's really possible to increase the spacing in the tabe. Any ideas?

[the remark this answers is not in the document]
Comment (mansnym) [reply src/components/CirclePacking.js]: Hehe nah not really, only thing I can come up with is to add additional whitespace but it seems a bit excessive. Everything else looks great so I think we merge this bad boy now

**Review by mansnym — APPROVED** [review]

**Pull request merged**